charter/index.html

```diff
@@ -253,11 +253,15 @@ <h3>
                             The PUMPKIN Protocol may include protocol details for integration with identity layers and mechanisms; access management and data integrity; notifications about resource changes; and authorization mechanisms.
                         </p>
                         <p>
-                          <b>Input document</b>: <a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a>, adopted from the <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>.
-                             <q lang="en-GB" cite="https://solidproject.org/TR/2022/protocol-20221231#introduction">An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralized web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.</q>
+                          <b>Input documents</b>:
                         </p>
+                        <ul>
+                          <li><a href="https://solidproject.org/ED/protocol">Solid Protocol, latest published version (0.10.0) or Editor's Draft</a>, adopted from <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a>.
+                            “An overarching design goal of the Solid ecosystem is to be evolvable and to provide fundamental affordances for decentralized web applications for information exchange in a way that is secure and privacy respecting. In this environment, actors allocate identifiers for their content, shape and store data where they have access to, set access controls, and use preferred applications and services to achieve them.”
+                          <li><a href="https://fcrepo.github.io/fcrepo-specification/">Fedora API Specification 1.0</a>.
+                        </ul>
                         <p class=new>
-                          The Working Group will consider other input, in addition to this specification, if it deems it relevant to its scope and to the goal of this deliverable.
+                          The Working Group will consider other input, in addition to these specifications, if it deems it relevant to its scope and to the goal of this deliverable.
```

Comment:
```suggestion
                          The Working Group will consider other input, in addition to these specifications, if the Working Group deems that input to be within the scope of its charter and relevant to the goals of its deliverables.
```

Reply:
I created a PR specific to this sentence (#74) so resolving this suggestion for now.